sdks/android/Mentat/library/src/main/java/org/mozilla/mentat/TupleResult.java

```diff
@@ -88,8 +85,7 @@ public Long asEntid(Integer index) {
      * @return  The keyword at that index.
      */
     public String asKeyword(Integer index) {
-        this.validate();
-        return JNA.INSTANCE.value_at_index_into_kw(this.rawPointer, index);
+        return getAndConsumeMentatString(JNA.INSTANCE.value_at_index_into_kw(this.validPointer(), index));
```

Comment:
(off-topic) this seems pretty brittle to me - we're implicitly relying on presence of a `clone()` in Rust to not crash when these kinds of methods are called more than once for a given index.